app/src/main/java/ru/vssemikoz/newsfeed/utils/Mappers.java

```diff
@@ -0,0 +1,22 @@
+package ru.vssemikoz.newsfeed.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import retrofit2.Response;
+import ru.vssemikoz.newsfeed.models.Category;
+import ru.vssemikoz.newsfeed.models.NewsApiResponse;
+import ru.vssemikoz.newsfeed.models.NewsApiResponseItem;
+import ru.vssemikoz.newsfeed.models.NewsItem;
+
+public class Mappers {
```

Comment:
Can a class with such a name follow SOLID principles? What are you going to place here?